test/runtime/Policies.go

```diff
@@ -669,6 +671,239 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		connectivityTest(allRequests, helpers.App2, helpers.Httpd1, BeTrue)
 	})
 
+	It("Checks CIDR Policy", func() {
+
+		ipv4Host := "192.168.254.254"
+		ipv4OtherHost := "192.168.254.111"
+		ipv4OtherNet := "99.11.0.0/16"
+		ipv6Host := "fdff::ff"
+		httpd2Label := "id.httpd2"
+		httpd1Label := "id.httpd1"
+
+		// Helper function which creates a file containing policyText which is
+		// assumed to be a valid Cilium Network Policy, imports it, and waits for
+		// the policy to be realized within Cilium. Deletes the created file when
+		// done.
+		renderAndImportPolicy := func(policyText string) {
+			policyFileName := fmt.Sprintf("policy_%s.json", helpers.MakeUID())
+			err := helpers.RenderTemplateToFile(policyFileName, policyText, 0777)
+			Expect(err).Should(BeNil())
+			path := helpers.GetFilePath(policyFileName)
+			defer os.Remove(policyFileName)
+			_, err = vm.PolicyImportAndWait(path, helpers.HelperTimeout)
+			Expect(err).Should(BeNil())
```

Comment:
assert message here please

Reply:
As I inadvertently duplicated functionality that already existed in `vm.PolicyRenderAndImport`, this will be removed in next upload.